### PR TITLE
chore(infrastructure): add MakeFile and prepare-commit-msg hook

### PR DESCRIPTION
**Pull request by SofianeBenziane**
## What does this PR do?

Simplify the implementation of the project with MakeFile and adding git hook.

## Tasks

Added MakeFile with command 'app-install' to initialize the project and configure git to use hooks.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+###############
+
+## initialize a git config to use hooks
+init-hooks:
+	git config --local core.hooksPath .githooks
+
+## to install app
+app-install:
+	npm install
+	make init-hooks
```
Comment (so-mouga) [inline Makefile]: Please add a blank line under this

Comment (so-mouga) [reply Makefile]: Do you test to commit with an other message like the regex ?
@Sbenziane  @Arnaud-Adon

Comment (SofianeBenziane) [reply Makefile]: I just added an blank line and yes, I have tested commit with message that doesn't match with the regex.

**Review by so-mouga — COMMENTED** [review]
ok

**Review by so-mouga — APPROVED** [review]
ok

**Pull request merged**